src/Microsoft.AspNet.Razor/TagHelpers/TagMode.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Razor.TagHelpers
```

Comment:
As discussed offline, we've got a few choices and none of them are perfect
1. Leave as-is which leads to additional `using` statements in tag helper classes that need to specify the requested structure or override the final mode.
2. Move to the `Runtime` namespace which creates a strange overlap between this project and the `Runtime` project.
3. Create separate `enum` declarations in the `Runtime` project and namespace. Might need a couple of casts to convert one into the other. But more importantly leads to code duplication, fortunately of code that should almost never change.